### ISSUE TITLE
Gif attachment autoplaying despite autoplay setting turned off

### ISSUE THREAD

**Issue by Fidgetcetera** [vector-im/element-web#15790]
### Description

The "Autoplay GIFs and videos" option turned off does nothing for attached, encrypted images.

### Steps to reproduce

- Be in encrypted chat.
- Receive an animated gif.
- Turn off autoplay.
- Gif still autoplays.

Gifs shouldn't autoplay when the setting is turned off.

Logs being sent: yes

![Screen capture gif of a gif in Element. It always plays, with the word "gif" sitting overtop of it. When the cursor enters the gif, the label disappears, and when the cursor enters or leaves, the gif resets to the start.](https://user-images.githubusercontent.com/15249812/99881718-5cd88480-2be9-11eb-8158-30cfb78e30c8.gif)

### Version information

- **Platform**: Desktop.

For the desktop app:

- **OS**: Xubuntu 20.04
- **Version**: 1.7.13

**Comment by anoadragon453** [comment]
If you're using [matrix-media-repo](https://github.com/turt2live/matrix-media-repo/) for your homeserver and have enabled animated thumbnails then this is [a known problem](https://github.com/vector-im/element-web/issues/11519).

**Comment by Fidgetcetera** [comment]
> If you're using [matrix-media-repo](https://github.com/turt2live/matrix-media-repo/) for your homeserver and have enabled animated thumbnails then this is [a known problem](https://github.com/vector-im/element-web/issues/11519).

Just checked with my admin, she hasn't enabled anything extra for media.

**Comment by Phenitei** [comment]
Hi, dropping in because of the Cannot Reproduce tag. I also have the same problem, am admin of my own server and can confirm that I am not using matrix-media-repo. Using Version `1.7.24` on the Desktop platform on Arch Linux.

**Comment by Mikaela** [comment]
This also affects avatars and is a violation of [Web Content Accessibility Guidelines 2.1 section 2.2.2 Pause, Stop, Hide](https://www.w3.org/TR/WCAG21/#pause-stop-hide). How this is an accessibility issue is explained in [Understanding Success Criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html).

I think one of the suggestions here would be this autoplay setting, [Setting animated gif images to stop blinking after n cycles (within 5 seconds)](https://www.w3.org/WAI/WCAG21/Techniques/general/G152).

**Comment by t3chguy** [comment]
@Mikaela this issue is strictly about attachments, for avatars please open a different issue. They function is rather different ways internally.

**Comment by Fidgetcetera** [comment]
Dropping in to say that whatever happened here, it seems to not be happening anymore on 1.7.34. Not sure if @Phenitei can still reproduce the issue, but if not, we probably should close it.

**Comment by Phenitei** [comment]
Hi, @Fidgetcetera is correct - the issue seems to have been resolved, one way or the other. This issue can be closed :)

**Comment by Fidgetcetera** [comment]
Great, closing!
